cropper/src/main/java/com/canhub/cropper/CropImageView.kt

```diff
@@ -1735,164 +1738,159 @@ class CropImageView @JvmOverloads constructor(context: Context, attrs: Attribute
-                    )
-                    options.guidelinesColor = ta.getInteger(
-                        R.styleable.CropImageView_cropGuidelinesColor,
-                        options.guidelinesColor
-                    )
-                    options.backgroundColor = ta.getInteger(
-                        R.styleable.CropImageView_cropBackgroundColor,
-                        options.backgroundColor
-                    )
-                    options.showCropOverlay = ta.getBoolean(
-                        R.styleable.CropImageView_cropShowCropOverlay,
-                        mShowCropOverlay
-                    )
-                    options.showProgressBar = ta.getBoolean(
-                        R.styleable.CropImageView_cropShowProgressBar,
-                        mShowProgressBar
-                    )
-                    options.borderCornerThickness = ta.getDimension(
-                        R.styleable.CropImageView_cropBorderCornerThickness,
-                        options.borderCornerThickness
-                    )
-                    options.minCropWindowWidth = ta.getDimension(
-                        R.styleable.CropImageView_cropMinCropWindowWidth,
-                        options.minCropWindowWidth.toFloat()
-                    ).toInt()
-                    options.minCropWindowHeight = ta.getDimension(
-                        R.styleable.CropImageView_cropMinCropWindowHeight,
-                        options.minCropWindowHeight.toFloat()
-                    ).toInt()
-                    options.minCropResultWidth = ta.getFloat(
-                        R.styleable.CropImageView_cropMinCropResultWidthPX,
-                        options.minCropResultWidth.toFloat()
-                    ).toInt()
-                    options.minCropResultHeight = ta.getFloat(
-                        R.styleable.CropImageView_cropMinCropResultHeightPX,
-                        options.minCropResultHeight.toFloat()
-                    ).toInt()
-                    options.maxCropResultWidth = ta.getFloat(
-                        R.styleable.CropImageView_cropMaxCropResultWidthPX,
-                        options.maxCropResultWidth.toFloat()
-                    ).toInt()
-                    options.maxCropResultHeight = ta.getFloat(
-                        R.styleable.CropImageView_cropMaxCropResultHeightPX,
-                        options.maxCropResultHeight.toFloat()
-                    ).toInt()
-                    options.flipHorizontally = ta.getBoolean(
-                        R.styleable.CropImageView_cropFlipHorizontally, options.flipHorizontally
+        val intent = (context as? Activity)?.intent
+        val options: CropImageOptions = intent?.let {
+            val bundle = it.getBundleExtra(CropImage.CROP_IMAGE_EXTRA_BUNDLE)
+            bundle?.getParcelable(CropImage.CROP_IMAGE_EXTRA_OPTIONS) ?: CropImageOptions()
+        } ?: CropImageOptions()
+        if (attrs != null) {
```

Comment:
Before this only happened when `options` was null, meaning, no bundle info.

Not this can happen even if the bundle has values.

Reply:
So what to do when the bundle is null...

Reply:
if the bundle is null we keep the same behaviour before.

We use `CropImageOptions()` values and `if (attrs != null) {` 

the issue, is that when bundle is not null we should **not** do `if (attrs != null) {`

Reply:
Oh I didn't noticed that